fuse/dl/models/backbones/backbone_resnet.py

```diff
@@ -47,9 +50,16 @@ def __init__(self, pretrained: bool = False, in_channels: int = 3, name: str = "
 
         # load pretrained parameters if required
         if pretrained:
+            print("Warning: not supported by new torchvision version - use weights instead")
+            from torch.hub import load_state_dict_from_url
+            from torchvision.models.resnet import model_urls
+
             state_dict = load_state_dict_from_url(model_urls[name])
             self.load_state_dict(state_dict)
 
+        if weights is not None:
```

Comment:
I suggest to add the following to make sure the user don't supply the two arguments (because one overrides the other)

```python
if weights and pretrained :
  raise Exception("Use only one method to load pre-trained weights. Two were given!")
```

(or maybe another msg 😅 )